src/System.Numerics.Vectors/src/System/Numerics/Matrix4x4.cs

```diff
@@ -3,12 +3,18 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Globalization;
+using System.Runtime.InteropServices;
+#if netcoreapp
```

Comment:
This is currently not "working" as expected. I don't see `netcoreapp` in the `DefineConstants` list